### PR TITLE
Updated to 1.9.0 release

### PR DESCRIPTION
**Pull request by undrfined**
- [x] Fix crashes on android and ios
- [x] Check entities
image
- [x] Check new 1.9 blocks
image
image

### CODE REVIEW

**Comment by Lind0f** [comment]
@NiclasOlofsson merge it?

File: src/MiNET/MiNET/Net/MCPE Protocol Documentation.md [hunk 1]
```diff
@@ -2330,8 +2330,8 @@ Wiki: [Update Soft Enum Packet](https://github.com/NiclasOlofsson/MiNET/wiki//Pr
 ### Network Stack Latency Packet (0x73)
 Wiki: [Network Stack Latency Packet](https://github.com/NiclasOlofsson/MiNET/wiki//Protocol-NetworkStackLatencyPacket)
 
-**Sent from server:** true  
-**Sent from client:** false
+**Sent from server:** false
```
Comment (dktapps) [inline src/MiNET/MiNET/Net/MCPE Protocol Documentation.md]: this is sent from both.

**Comment by endervlad** [comment]
When I join with android - game crashes(

**Comment by NiclasOlofsson** [comment]
Already merged from other PR.

**Pull request closed without merging**